blueocean-dashboard/src/main/js/components/karaoke/components/Step.jsx

```diff
@@ -169,11 +187,13 @@ export class Step extends Component {
 }
 
 Step.propTypes = {
-    augmenter: PropTypes.object,
+    pipelineView: PropTypes.object,
```

Comment:
Probably this would be ideal to move `liveUpdate` and a list of expanded stuff (e.g. `steps`) into the `pipelineView` to simplify some of this